features/steps/consistency.py

```diff
@@ -73,11 +73,11 @@ def step_impl(context, image, output_format):
     if output_format == 'COLOR_PROPHOTO_RGB':
         output_switch = '-color ProPhotoRGB'
     output_split = output_switch.split(' ')
-    # there's probably a more pythonic way to do this, with array flattening etc.
-    # just being expedient for the time being
-    args = [found_executable, '-tiff', output_split[0], image]
-    if len(output_split) == 2:
-        args = [found_executable, '-tiff', output_split[0], output_split[1], image]
+    if output_format != 'CROP':
+        output_split = output_split + ['-no-crop']
+    if output_format == 'CROP':
+        output_split = []
```

Comment:
Kind of redundant do `output_split = []` here . If you want to do that you could at least use `else:`